sympy/functions/elementary/hyperbolic.py

```diff
@@ -771,7 +771,13 @@ def _sage_(self):
 ############################# HYPERBOLIC INVERSES #############################
 ###############################################################################
 
-class asinh(Function):
+class InverseHyperbolicFunction(Function):
+    """Base class for inverse hyperbolic functions."""
+
+    pass
```

Comment:
@smichr This was for #13110 .
Since it is closed, it is unnecessary for now, but I put and left it here for future usage in simplification, manipulation, etc.